### PR TITLE
feat: implementation af History for organization

### PR DESCRIPTION
**Pull request by FirestarJes**

### CODE REVIEW

File: source/marketparticipant/Energinet.DataHub.MarketParticipant.Application/Commands/Organization/OrganizationAuditLogDto.cs [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using Energinet.DataHub.MarketParticipant.Domain.Model;
+
+namespace Energinet.DataHub.MarketParticipant.Application.Commands.Organization;
+
+public record OrganizationAuditLogDto(
+    OrganizationId OrganizationId,
```
Comment (defectiveAi) [inline source/marketparticipant/Energinet.DataHub.MarketParticipant.Application/Commands/Organization/OrganizationAuditLogDto.cs:21]: Return Guid for OrganizationId, not domain model.

File: source/marketparticipant/Energinet.DataHub.MarketParticipant.Infrastructure/Persistence/EntityConfiguration/OrganizationEntityConfiguration.cs [hunk 2]
```diff
@@ -13,21 +13,28 @@
...
     {
-        public void Configure(EntityTypeBuilder<OrganizationEntity> builder)
+        protected override void ConfigureEntity(EntityTypeBuilder<OrganizationEntity> builder)
         {
             ArgumentNullException.ThrowIfNull(builder, nameof(builder));
             builder.ToTable("Organization");
             builder.HasKey(organization => organization.Id);
             builder.Property(organization => organization.Id).ValueGeneratedOnAdd();
-            builder.OwnsOne(organization => organization.Address);
+            builder.Property("PeriodStart").HasColumnName("PeriodStart");
```
Comment (defectiveAi) [inline source/marketparticipant/Energinet.DataHub.MarketParticipant.Infrastructure/Persistence/EntityConfiguration/OrganizationEntityConfiguration.cs:31]: PeriodStart and PeriodEnd should not be necessary.

Comment (FirestarJes) [reply source/marketparticipant/Energinet.DataHub.MarketParticipant.Infrastructure/Persistence/EntityConfiguration/OrganizationEntityConfiguration.cs:31]: You are right, they were part of what i tested, i'll remove them

File: source/marketparticipant/Energinet.DataHub.MarketParticipant.Application/Commands/Organization/OrganizationAuditLogDto.cs (hunk 1 above)
Comment (FirestarJes) [reply source/marketparticipant/Energinet.DataHub.MarketParticipant.Application/Commands/Organization/OrganizationAuditLogDto.cs]: Fixed

**Review by defectiveAi — APPROVED** [review]

**Pull request merged**